### PR TITLE
Fund 7 challenge Translations - English

### PR DESCRIPTION
**Pull request by PardusEidolon**
# Fund 7 challenge Translations - `English`

This is here for the English markdown files for all the Fund 7 challenges to be translated into other languages. when these are complete translations on these documents can begin using the English markdown file as a reference for translators.

You can refer here #48 to keep track of both the challenges and proposals.

### CODE REVIEW

**Comment by wolstaeb** [comment]
The **Guiding Questions** section of the **F7: Catalyst Natives COTI: Pay with ADA Plug-in** challenge is missing and needs to be added.

**Review by wolstaeb — APPROVED** [review]

File: content/en/catalyst/fund7/challenges/Accelerate Decentralized Identity/campaign-brief.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+title: "Accelerate Decentralized Identity"
+linkTitle: "Accelerate Decentralized Identity"
+type: docs
+hide_feedback: true
+---
+How to make the auditing process of funded proposals efficient, distributed and transparent?
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Accelerate Decentralized Identity/campaign-brief.md:7]:
```suggestion
What solutions can we develop and implement with Atala PRISM and Cardano today to have the most positive impact and open up opportunities for rapid growth of the Cardano network
```

File: content/en/catalyst/fund7/challenges/Accelerate Decentralized Identity/campaign-brief.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+New legislation in Wyoming allows for a decentralized autonomous organization (DAO) to be registered with the state. We challenge the community to establish a DAO under the new laws. It could manage an impact or development goal with shared common resources governed by the DAO.
+
+We are curious to see if our community proposes a DAO. With a funded proposal, we could facilitate introductions to legal experts in Wyoming.
+
+The proposal should consider setting aside funds for legal expenses to establish the DAO and legal documentation.
+
+##### Patterns of Commoning
+
+We draw inspiration from the work of the Trust Over IP Foundation and the principles from 'Patterns of Commoning.' We see the potential for innovation by linking communities with shared interests under these principles.
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Accelerate Decentralized Identity/campaign-brief.md:39]:
```suggestion
We draw inspiration from the work of the [Trust Over IP Foundation](https://wiki.trustoverip.org/display/HOME/YOMA+Ecosystem+Task+Force) and the principles from 'Patterns of Commoning.' We see the potential for innovation by linking communities with shared interests under these principles.
```

File: content/en/catalyst/fund7/challenges/Accelerate Decentralized Identity/campaign-brief.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+On the other side, distribution or demand-driven commerce has the potential to build a market force from the bottom-up that can engage the industrial global supply chain in ways that are not possible without establishing the voice and economic strength of the community digitally.
+
+For the supply chain challenge, we want to address two significant challenges:
+
+##### Small Producer Problem
+
+How can we create transparency as the bottom-up process meets the industrialized supply chains, cuts through the centralized distribution markets, and connects directly to the consumer?
+
+For example, small producers like a family ranch in Wyoming have no way to get information about the value of their product through these global aggregation and distribution networks. Currently, a solution such as the BeefChain project is supporting small ranchers.
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Accelerate Decentralized Identity/campaign-brief.md:73]:
```suggestion
For example, small producers like a family ranch in Wyoming have no way to get information about the value of their product through these global aggregation and distribution networks. Currently, a solution such as the [BeefChain project](https://beefchain.com/) is supporting small ranchers.
```

Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Accelerate Decentralized Identity/campaign-brief.md:73]:
```suggestion
For example, small producers like a family ranch in Wyoming have no way to get information about the value of their product through these global aggregation and distribution networks. Currently, a solution such as the [BeefChain project](https://beefchain.com/) is supporting small ranchers.
```

File: content/en/catalyst/fund7/challenges/Catalyst Accelerator and Mentors/campaign-brief.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+## How does success look like?
+- Access to financial, human, and intellectual capital to fuel the success of Catalyst-funded projects
+- Programmes that share in the spirit of co-creation, willing to collaborate with us at Catalyst, Cardano stakeholders, and the Catalyst cohorts.
+
+## Key Metrics to measure
+- Number of projects that can be accelerated in the scope of the funding
+- Number of mentors and investors that can be onboarded into the ecosystem
+- Number of mentors and investors that can be onboarded into the ecosystem
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Catalyst Accelerator and Mentors/campaign-brief.md:41]:
```suggestion
- Distribution of programmes across the world to serve different audiences
```

File: content/en/catalyst/fund7/challenges/Grow Latin America, Grow Cardano/campaign-brief.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Grow Latin America, Grow Cardano"
+linkTitle: "Grow Latin America, Grow Cardano"
+type: docs
+hide_feedback: true
+---
+How do we grow the Cardano ecosystem in Latin America?
+Fund Size: **$500,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Grow Latin America, Grow Cardano/campaign-brief.md:8]:
```suggestion

Fund Size: **$500,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Global Sustainable Indepependant-SPO's/campaign-brief.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Global Sustainable Indep. SPO's"
+linkTitle: "Global Sustainable Indep. SPO's"
+type: docs
+hide_feedback: true
+---
+How can we promote a decentralised, globally distributed and sustainable pool of operators to run the Cardano network
+Fund Size: **$100,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Global Sustainable Indepependant-SPO's/campaign-brief.md:8]:
```suggestion

Fund Size: **$100,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Catalyst Accelerator and Mentors/campaign-brief.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+---
+title: " Catalyst Accelerator & Mentors"
+linkTitle: " Catalyst Accelerator & Mentors"
+type: docs
+hide_feedback: true
+---
+How can Catalyst harness professional firepower to bootstrap an accelerator environment that amplifies and commercializes funded projects?
+
+
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Catalyst Accelerator and Mentors/campaign-brief.md:9]:
```suggestion
```

File: content/en/catalyst/fund7/challenges/Mini-Low-Budget Dapps and Integrations/campaign-brief.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Mini/Low-Budget Dapps & Integrations"
+linkTitle: "Mini/Low-Budget Dapps & Integrations"
+type: docs
+hide_feedback: true
+---
+What Low Budget / Mini Decentralized Applications will provide the most value for end users in the next 3 months?
+Fund Size: **$200,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Mini-Low-Budget Dapps and Integrations/campaign-brief.md:8]:
```suggestion

Fund Size: **$200,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Miscellaneous Challenge/campaign-brief.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Miscellaneous Challenge"
+linkTitle: "Miscellaneous Challenge"
+type: docs
+hide_feedback: true
+---
+How do we create a space for ideas and projects that do not fit any of the other current Challenges?
+Fund Size: **$200,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Miscellaneous Challenge/campaign-brief.md:8]:
```suggestion

Fund Size: **$200,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Nation Building Dapps/campaign-brief.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+## Campaign Brief
+Dapp proposals should cover foundational Registry solutions, namely:
+
+National ID & Civil Registry, Property Registration, Medical & Vaccination, Education Census, Revenue Service, Business Incorporations, Elections Voting, Customs & Border Control, Criminal Records, Employment Census, Vehicle Registration, Licensing & Certifications.
+
+Proposers to review IOHK/ EMURGO Atala solutions before hand along with reference use cases on integrated registry systems applications currently implemented across wealthy nations.
+
+It is further advised Proposers to obtain a knowledge base over research done in the past three decades unraveling the "mysterious" causes of poverty in third world countries by the economist Hernando de Soto. The attached book "The Mystery of Capital" is a great reference to get into context for anyone involved in developing proposals for this challenge.
+
+Introduction video:
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Nation Building Dapps/campaign-brief.md:22]:
```suggestion
[Introduction video](https://archive.org/details/Hernando_De_Soto_The_Mystery_Of_Capital_Why_Capitalism_TriumphsIn_The_West_And_Fails_Everywhere_Else)
```

File: content/en/catalyst/fund7/challenges/Nation Building Dapps/campaign-brief.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+https://archive.org/details/Hernando_De_Soto_The_Mystery_Of_Capital_Why_Capitalism_TriumphsIn_The_West_And_Fails_Everywhere_Else
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Nation Building Dapps/campaign-brief.md:24]:
```suggestion
```

File: content/en/catalyst/fund7/challenges/New SPO Business Opportunities/campaign-brief.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
+---
+title: "New SPO Business Opportunities"
+linkTitle: "New SPO Business Opportunities"
+type: docs
+hide_feedback: true
+---
+Stake Pool Operators are a key-success-factor of Cardano. With their infrastructure and capabilities, important new services can be created
+Fund Size: **$250,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/a/campaign-home/26246) and [browse ideas](https://cardano.ideascale.com/a/ideas/top/campaign-filter/byids/campaigns/26246/stage/unspecified).
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/New SPO Business Opportunities/campaign-brief.md:10]:
```suggestion
[Campaign page](https://cardano.ideascale.com/a/campaign-home/26245) and [browse ideas](https://cardano.ideascale.com/a/ideas/top/campaign-filter/byids/campaigns/26245/stage/unspecified).
```

File: content/en/catalyst/fund7/challenges/New SPO Business Opportunities/campaign-brief.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
+---
+title: "New SPO Business Opportunities"
+linkTitle: "New SPO Business Opportunities"
+type: docs
+hide_feedback: true
+---
+Stake Pool Operators are a key-success-factor of Cardano. With their infrastructure and capabilities, important new services can be created
+Fund Size: **$250,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/New SPO Business Opportunities/campaign-brief.md:8]:
```suggestion

Fund Size: **$250,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Open Source Developer Ecosystem/campaign-brief.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Open Source Developer Ecosystem"
+linkTitle: "Open Source Developer Ecosystem"
+type: docs
+hide_feedback: true
+---
+How can we ensure that the cardano ecosystem is build on a framework which is owned by the community and equally accessible to all?
+Fund Size: **$200,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Open Source Developer Ecosystem/campaign-brief.md:8]:
```suggestion

Fund Size: **$200,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Scale-UP Cardano's Community Hubs"
+linkTitle: "Scale-UP Cardano's Community Hubs"
+type: docs
+hide_feedback: true
+---
+How do we encourage Local Community Hubs to drive adoption and awareness of Cardano in the next 6 months?
+Fund Size: **$350,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md:8]:
```suggestion

Fund Size: **$350,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Scale-UP Cardano's Community Hubs"
+linkTitle: "Scale-UP Cardano's Community Hubs"
+type: docs
+hide_feedback: true
+---
+How do we encourage Local Community Hubs to drive adoption and awareness of Cardano in the next 6 months?
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md:7]:
```suggestion
How do we encourage Local Community Hubs to drive adoption and awareness of Cardano in the next 6 months?

![Scale-Up Cardano Community Hubs](https://i.imgur.com/bZQvrLI.jpeg)
```

File: content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+## Campaign Brief
+Out of the world's approximately 7.8 billion inhabitants, 6.45 billion do not speak English.
+
+Living in the Western world, we forget this fact regularly, and many of us have never even considered it.
+
+Let's consider it now and start to build an ecosystem that is truly global, allowing everyone to participate to the best of their ability.
+
+When you drive adoption at the ground level, you are doing more than just 'building cool products', you are empowering people to change their lives in ways they could have never imagined by themselves.
+
+https://imgur.com/a/zK5ralU
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md:21]:
```suggestion
```

Comment (wolstaeb) [reply content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md:21]:
```suggestion
```

File: content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md [hunk 18]
```diff
@@ -0,0 +1,76 @@
+---
+title: "Seeding Cardano's Grassroots DeFi"
+linkTitle: "Seeding Cardano's Grassroots DeFi"
+type: docs
+hide_feedback: true
+---
+How to encourage new teams to Deploy Defi Solutions on Cardano?
+Fund Size: **$200,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md:8]:
```suggestion

Fund Size: **$200,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md [hunk 19]
```diff
@@ -0,0 +1,76 @@
...
+
+With Catalyst, we have the opportunity to fund DeFi projects and allow them to have greater decentralization and resilience, which are essential aspects for DeFi projects. Many projects end up accepting the participation of Venture Capitals due to lack of financial resources. The participation of VCs in DeFi projects usually ends up causing a large part of the supply distribution to be alocated to VCs and this compromises the decentralization and consequently the security of these projects.
+
+The DeFi ecosystem today is taken over by projects full of vulnerabilities, hacks are constant, economic models are copied from other projects, decentralization is no longer a priority and few projects are dedicated to create economic models/bonding curves/algorithms thinking about the users' needs.
+
+It has become clear over the year that bugs, hacks and vulnerabilities are not unique to smart contracts on Ethereum. Dapps running on Binance Smart Chain and Cosmos suffered from vulnerabilities as well.
+
+https://cointelegraph.com/news/hackers-stole-at-least-600m-in-poly-exploit-across-three-chains
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md:27]:
```suggestion
- [Hackers stole at least $600M in Poly exploit across three chains](https://cointelegraph.com/news/hackers-stole-at-least-600m-in-poly-exploit-across-three-chains)
```

File: content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+
+https://www.theblockcrypto.com/post/111660/thorchain-suffers-5-million-exploit-developers-have-put-out-a-fix
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md:29]:
```suggestion
- [Thorchain suffers $5 million exploit, developers have put out a fix](https://www.theblockcrypto.com/post/111660/thorchain-suffers-5-million-exploit-developers-have-put-out-a-fix)
```

File: content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md [hunk 21]
```diff
@@ -0,0 +1,76 @@
...
+- Team B with 5 proposals of $10,000 USD
+- Team C with 1 proposal of $50,000 USD
+- Team D with 1 proposal of $15,000 USD + 1 proposal of $35,000 US
+
+## Why is it Important?
+So that new teams can have a kickstart in their projects without having to resort to ICOs and VCs allocation.
+
+
+## How does success look like?
+At least 4 new teams deploying DeFi dapps on Cardano in 2022 + Managing to form a community in order to gain competitiveness/susteinability
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Seeding Cardano's Grassroots DeFi/campaign-brief.md:61]:
```suggestion
At least 4 new teams deploying DeFi Dapps on Cardano in 2022 + Managing to form a community in order to gain competitiveness/sustainability
```

**Comment by wolstaeb** [comment]
Change the folder names so that they are simpler and do not include spaces and punctuation. For example:

```.../catalyst/fund7/challenges/A.I. & SingularityNet a $5T market/campaign-brief.md```

Should be changed to something like:

```.../catalyst/fund7/challenges/ai-singularitynet/campaign-brief.md```

The TOML frontmatter is used to name pages and where they sit in the sites hirrachy. Space and punctuations in file or folder names will just cause grief with tools like hugo or gitlocalize.

File: content/en/catalyst/fund7/challenges/Boosting Cardano's DeFi/campaign-brief.md [hunk 22]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Boosting Cardano's DeFi"
+linkTitle: " Boosting Cardano's DeFi"
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Boosting Cardano's DeFi/campaign-brief.md:3]:
```suggestion
linkTitle: "Boosting Cardano's DeFi"
```

File: content/en/catalyst/fund7/challenges/Boosting Cardano's DeFi/campaign-brief.md [hunk 23]
```diff
@@ -0,0 +1,78 @@
...
+
+With Catalyst, we have the opportunity to fund DeFi projects and allow them to have greater decentralization and resilience, which are essential aspects for DeFi projects. Many projects end up accepting the participation of Venture Capitals due to lack of financial resources. The participation of VCs in DeFi projects usually ends up causing a large part of the supply distribution to be alocated to VCs and this compromises the decentralization and consequently the security of these projects.
+
+The DeFi ecosystem today is taken over by projects full of vulnerabilities, hacks are constant, economic models are copied from other projects, decentralization is no longer a priority and few projects are dedicated to create economic models/bonding curves/algorithms thinking about the users' needs.
+
+It has become clear over the year that bugs, hacks and vulnerabilities are not unique to smart contracts on Ethereum. Dapps running on Binance Smart Chain and Cosmos suffered from vulnerabilities as well.
+
+[https://cointelegraph.com/news/hackers-stole-at-least-600m-in-poly-exploit-across-three-chains]()
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Boosting Cardano's DeFi/campaign-brief.md:33]:
```suggestion
- [Hackers stole at least $600M in Poly exploit across three chains](https://cointelegraph.com/news/hackers-stole-at-least-600m-in-poly-exploit-across-three-chains)
```

File: content/en/catalyst/fund7/challenges/Boosting Cardano's DeFi/campaign-brief.md [hunk 24]
```diff
@@ -0,0 +1,78 @@
...
+
+[https://www.theblockcrypto.com/post/111660/thorchain-suffers-5-million-exploit-developers-have-put-out-a-fix]()
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Boosting Cardano's DeFi/campaign-brief.md:35]:
```suggestion
[Thorchain suffers $5 million exploit, developers have put out a fix](https://www.theblockcrypto.com/post/111660/thorchain-suffers-5-million-exploit-developers-have-put-out-a-fix)
```

File: content/en/catalyst/fund7/challenges/Catalyst - Rapid Funding Mechanisms/campaign-brief.md [hunk 25]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Catalyst - Rapid Funding Mechanisms"
+linkTitle: "Catalyst - Rapid Funding Mechanisms"
+type: docs
+hide_feedback: true
+---
+How can we distribute Cardano Treasury funds more rapidly in a transparent, accountable, and safe manner?
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Catalyst - Rapid Funding Mechanisms/campaign-brief.md:7]:
```suggestion
How can we distribute Cardano treasury funds more rapidly in a transparent, accountable, and safe manner?
```

File: content/en/catalyst/fund7/challenges/Community Events/campaign-brief.md [hunk 26]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Community Events"
+linkTitle: "Community Events"
+type: docs
+hide_feedback: true
+---
+How can we increase meaningful participation among the community in the next 6 months?
+
+Fund Size: **$500,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Community Events/campaign-brief.md:9]:
```suggestion
Fund Size: **$200,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Connecting Japan 日本 Community/campaign-brief.md [hunk 27]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Connecting Japan/日本 Community"
+linkTitle: "Connecting Japan/日本 Community"
+type: docs
+hide_feedback: true
+---
+How can we help the Japanese community to adopt Cardano and connect with the Catalyst community?
+Fund Size: **$50,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/a/campaign-home/26234) and [browse ideas](https://cardano.ideascale.com/a/ideas/top/campaign-filter/byids/campaigns/26234/stage/unspecified).
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Connecting Japan 日本 Community/campaign-brief.md:10]:
```suggestion
[Campaign page](https://cardano.ideascale.com/a/campaign-home/26238) and [browse ideas](https://cardano.ideascale.com/a/ideas/top/campaign-filter/byids/campaigns/26238/stage/unspecified).
```

File: content/en/catalyst/fund7/challenges/Connecting Japan 日本 Community/campaign-brief.md [hunk 28]
```diff
@@ -0,0 +1,22 @@
...
+
+## Why is it Important?
+There has been an interest in Cardano in Japan from the start and the country has had a progressive stance towards blockchain technology. This makes Japan a likely country to adopt solutions built on top of Cardano. There are already some active community members and SPO's but participation in Catalyst is still relatively low.
+
+Recently an Eastern Town Hall was started by community members and participation in the Japanese breakout room has been among the highest consistently. In the fund 6 challenges there are several proposals coming from the Japanese community as well. With the current momentum a challenge in fund 7 would likely see good participation and can solidify the community.
+
+At the moment most of Ideascale & Catalyst operates in english. This creates a barrier for all non-english communities. If solutions from this challenge prove successful in bridging that gap, they can function as an example for other communities.
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Connecting Japan 日本 Community/campaign-brief.md:22]:
```suggestion
At the moment most of Ideascale & Catalyst operates in english. This creates a barrier for all non-english communities. If solutions from this challenge prove successful in bridging that gap, they can function as an example for other communities.

## Possible directions:

- Translating key documentation
- Creating educational content about Cardano & Catalyst in Japanese
- Organising interpreters for Catalyst events
- Organising workshops for new members, CA's, SPO's, proposers
- Setting up community hubs
```

File: content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md [hunk 29]
```diff
@@ -0,0 +1,52 @@
+---
+title: "DApps - Integrations"
+linkTitle: "DApps - Integrations"
+type: docs
+hide_feedback: true
+---
+What applications will provide the most value for end users in 2022?
+Fund Size: **$250,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md:8]:
```suggestion

Fund Size: **$250,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md [hunk 30]
```diff
@@ -0,0 +1,52 @@
...
+
+[Campaign page](https://cardano.ideascale.com/a/campaign-home/26240) and [browse ideas](https://cardano.ideascale.com/a/ideas/top/campaign-filter/byids/campaigns/26240/stage/unspecified).
+
+## Campaign Brief
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md:12]:
```suggestion
```

File: content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md [hunk 31]
```diff
@@ -0,0 +1,52 @@
...
+*~ N/A*
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md:13]:
```suggestion
```

File: content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md (hunk 29 above)
Comment (wolstaeb) [reply content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md:8]:
```suggestion

Fund Size: **$250,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md [hunk 32]
```diff
@@ -0,0 +1,52 @@
...
+
+Let an "application output" and an "application transaction" be an outputs and transactions that use the applications' script addresses; and an "application token" be a token that's issued by the application.
+### Usage of applications developed during the challenge:
+- Number/growth of application output
+- Number/growth of application transactions
+- Total volume of Ada involved in application transactions
+- Total/growth of resource usage by applications
+- Number/growth of distinct addresses mentioned in application transactions
+
+### If the application uses tokens:
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/DApps - Integrations/campaign-brief.md:33]:
```suggestion
If the application uses tokens:
```

File: content/en/catalyst/fund7/challenges/Disarm Cyber Disinformation Attacks/campaign-brief.md [hunk 33]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Disarm cyber disinformation attacks"
+linkTitle: "Disarm cyber disinformation attacks"
+type: docs
+hide_feedback: true
+---
+How can we prevent attempts to destabilize Cardano through Distributed Denial of Truth (DDoT) attack scenarios?
+Fund Size: **$100,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Disarm Cyber Disinformation Attacks/campaign-brief.md:8]:
```suggestion

Fund Size: **$100,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Disarm Cyber Disinformation Attacks/campaign-brief.md [hunk 34]
```diff
@@ -0,0 +1,64 @@
...
+
+Disinformation is fabricated or deliberately manipulated audio/visual/text content, intentionally created conspiracy theories or rumors or to sow mistrust. With disinformation the bad information is spread with the intent to deceive.
+
+Malinformation is the deliberate publication of information for personal or corporate rather than public interest. Deliberate change of context, date or time of genuine content.
+
+These DDoT attack scenarios may emerge as a combined outside/inside attack and spread faster in fiercely tribal communities, such as a cryptocurrency community. A DDoT attack might overwhelm a system with false information, leading to a plunge in confidence / community split.
+
+"Right now everybody is implicitly assuming that the only possible victim is an election," says David Perlman, a former Twitter Inc. data scientist who is now developing ways for corporations to counter disinformation at the computer security company Leviathan Security Group Inc. "There's no reason that a company couldn't be a victim," he says.*
+
+https://www.wsj.com/articles/brands-face-a-new-online-threat-disinformation-attacks-11602187365
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Disarm Cyber Disinformation Attacks/campaign-brief.md:25]:
```suggestion
- [Brands Face a New Online Threat: Disinformation Attacks](https://www.wsj.com/articles/brands-face-a-new-online-threat-disinformation-attacks-11602187365)
```

File: content/en/catalyst/fund7/challenges/Gamers On-Chained/campaign-brief.md [hunk 35]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Gamers On-Chained"
+linkTitle: "Gamers On-Chained"
+type: docs
+hide_feedback: true
+---
+What Blockchain gaming applications should be developed to achieve mass adoption of Cardano?
+Fund Size: **$250,000 USD paid in Ada**
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Gamers On-Chained/campaign-brief.md:8]:
```suggestion

Fund Size: **$250,000 USD paid in Ada**
```

File: content/en/catalyst/fund7/challenges/Lobbying for favorable legislation/campaign-brief.md [hunk 36]
```diff
@@ -0,0 +1,38 @@
...
+Fund Size: **$75,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/a/campaign-home/26235) and [browse ideas](https://cardano.ideascale.com/a/ideas/top/campaign-filter/byids/campaigns/26235/stage/unspecified).
+
+## Campaign Brief
+The idea behind this challenge is to fund specialists in regulation, lawyers and politicians who can promote the creation of an environment of laws that are more favorable or less harmful to the crypto space, with a focus on PoS and on Cardano's particularities.
+
+Through this challenge, we can fund proposals that seek the adoption of Cardano in certain jurisdictions, as happened recently with Bitcoin in El Salvador.
+
+https://www.coindesk.com/policy/2021/09/07/bitcoin-now-legal-tender-in-el-salvador-marking-world-first/
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Lobbying for favorable legislation/campaign-brief.md:18]:
```suggestion
- [Bitcoin Now Legal Tender in El Salvador, Marking World First](https://www.coindesk.com/policy/2021/09/07/bitcoin-now-legal-tender-in-el-salvador-marking-world-first/)
```

File: content/en/catalyst/fund7/challenges/Lobbying for favorable legislation/campaign-brief.md [hunk 37]
```diff
@@ -0,0 +1,38 @@
...
+
+Lobby and educate rational politicians so that it becomes possible to plead for the creation of more favorable rules on taxation, use of cryptocurrencies and greater privacy for cryptocurrency users.
+
+In August 2021, the US Senate voted for unforgiving legislation on the crypto ecosystem. We have few voices to defend the our community in the political class. Initiatives like the Coin Center are important to defend our community. How can we defend ourselves from attacks by authoritarian governments and compromising legislation?
+
+https://www.vice.com/en/article/4avpp3/cryptocurrency-lost-big-in-senate-infrastructure-bill-provision-now-what
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Lobbying for favorable legislation/campaign-brief.md:24]:
```suggestion
[Cryptocurrency Lost Big In Senate Infrastructure Bill Provision. Now What?](https://www.vice.com/en/article/4avpp3/cryptocurrency-lost-big-in-senate-infrastructure-bill-provision-now-what)
```

File: content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md [hunk 38]
```diff
@@ -0,0 +1,35 @@
...
+
+https://imgur.com/a/feXUVDF
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/Scale-UP Cardano's Community Hubs/campaign-brief.md:23]:
```suggestion
```

**Review by wolstaeb — CHANGES_REQUESTED** [review]
I've corrected or added missing pieces, links, and corrected some of the markdown formating. These changes have been batched.

**Review by wolstaeb — APPROVED** [review]

File: content/en/catalyst/fund7/challenges/accelerate-decentralized-id/campaign-brief.md [hunk 39]
```diff
@@ -0,0 +1,132 @@
+---
+title: "Accelerate Decentralized Identity"
+linkTitle: "Accelerate Decentralized Identity"
+type: docs
+hide_feedback: true
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/accelerate-decentralized-id/campaign-brief.md:5]:
```suggestion
hide_feedback: true
description: >
What solutions can we develop and implement with Atala PRISM and Cardano today to have the most positive impact and open up opportunities for the rapid growth of the Cardano network?
```

File: content/en/catalyst/fund7/challenges/accelerate-decentralized-id/campaign-brief.md [hunk 40]
```diff
@@ -0,0 +1,132 @@
...
+---
+What solutions can we develop and implement with Atala PRISM and Cardano today to have the most positive impact and open up opportunities for rapid growth of the Cardano network
```
Comment (wolstaeb) [inline content/en/catalyst/fund7/challenges/accelerate-decentralized-id/campaign-brief.md:7]:
```suggestion
What solutions can we develop and implement with Atala PRISM and Cardano today to have the most positive impact and open up opportunities for the rapid growth of the Cardano network?
```

File: content/en/catalyst/fund7/challenges/A.I. & SingularityNet a $5T market/campaign-brief.md [hunk 41]
```diff
@@ -0,0 +1,72 @@
+---
+title: "A.I. & SingularityNet a $5T market"
+linkTitle: "A.I. & SingularityNet a $5T market"
+type: docs
+hide_feedback: true
```
Comment (PardusEidolon) [inline content/en/catalyst/fund7/challenges/A.I. & SingularityNet a $5T market/campaign-brief.md]:
```suggestion
hide_feedback: true
description: >
How can we create a mutually beneficial partnership with SingularityNet and other A.I. developers and researchers?
```

**Review by wolstaeb — APPROVED** [review]

**Pull request merged**